proxylib/test/accesslog_server.go

```diff
@@ -26,6 +27,7 @@ type AccessLogServer struct {
 	Logs     chan cilium.EntryType
 	closing  uint32 // non-zero if closing, accessed atomically
```

Comment:
Yes, I had the thought, but I am not familiar at all with this code, so I figured I'd leave it for now. But since I have to revise anyways, I will take another look at this.

Reply:
Decided to use a channel.